content/influxdb/v2.4/install.md

```diff
@@ -86,6 +86,20 @@ do the following:
     If you do not move the `influxd` binary into your `$PATH`, prefix the executable
     `./` to run it in place.
 
+{{< expand-wrapper >}}
+{{% expand "<span class='req'>Recommended</span> – Set appropriate directory permissions" %}}
+
+To prevent unwanted access to data, we recommend setting the permissions on the influxdb `data-dir` to not be world readable.
+
+Example:
+
+```shell
+> chmod 0750 ~/.influxdbv2
+```
+
+{{% /expand %}}
+{{< /expand-wrapper >}}
```

Comment:
@jeffreyssmith2nd You can put two `expand` blocks in one `expand-wrapper` block. Instead of having two separate `expand-wrappers` here, would you move the `expand` block into the one below, or vice versa:

```html
{{< expand-wrapper >}}
{{% expand "<span class='req'>Recommended</span> – Set appropriate directory permissions" %}}

<!-- Directory permissions content -->

{{% /expand %}}
{{% expand "<span class='req'>Recommended</span> – Verify the authenticity of downloaded binary" %}}

<!-- Binary authenticity content -->

{{% /expand %}}
{{< /expand-wrapper >}}
```